### ISSUE TITLE
Implement Stripe

### ISSUE THREAD

**Issue by vpetersson** [WoTTsecurity/api#542]
**Tl;dr:** We need to start accepting payments through Stripe.

## Pricing/Tiers

| Tier        | Description           | Limitations  | Price |
| ------------- |-------------| ----- | - | 
| Free  | Entry-level tier | 1 node | $0 |
| Standard | This will be the regular plan that we will focus on  |   No upper limit | $9 per node/month | 
| Enterprise | This is a custom plan for large customers  |    Custom | Custom | 

### Considerations

 * The standard plan should have a 30 day free trial. This trial should require a credit card on file.
 * The standard plan includes the one free node. That is, if you have two nodes, you would only be charged for one because of the free tier.
 * We should use Stripe's widget to create accepting the credit card on sign-up.
 * Ignore all existing accounts and just focus on new users for now.
 * We need to use a [subscription](https://stripe.com/docs/billing/subscriptions/set-up-subscription) such that users are automatically billed.
 * We should push as much data as we can to Stripe (i.e. name/company/email)
 * We should evaluate using a library like [dj-stripe](https://github.com/dj-stripe/dj-stripe) to speed up things.
 
## Signup-page

The signup page will require the user to select the plan. We should not require a credit card for the free tier.

We should also not expose the enterprise tier here, as that is only for the pricing page where we will have a 'Contact us' form.

## Profile-page

The profile page need to have a new Billing section with some key features:

 * Allow Free us upgrade to Standard (ask for CC etc)
 * Allow users to cancel their plan

### Moving from Free to Standard

If a free tier user tries to add a secondary node, a warning need to appear telling him/her that he needs to upgrade to the standard tier. This should then be linked to the billing tab in the profile page.

### Cancellations

If a user cancels a plan, all his/her devices need to enter into an 'Inactive' state where he/she cannot see any information about them other than the name. Recommendations should also be dimmed out.

### Enterprise

Enterprise plans will be uncapped and unlikely managed through Stripe. We just need to have this tier in the backend such that we can manually map  a user.
